.gitignore

```diff
@@ -1,3 +1,14 @@
 __pycache__
 docs/build
-/docs/source/_autosummary/
\ No newline at end of file
+/docs/source/_autosummary/
+
+# Python files
+*.pyc
+__pycache__/
+
+# Exclude non-Python files from Black formatting

```

Comment:
.gitignore should be used like this

Reply:
black should ignore these files because they are not python files, I don't see why you are listing them here

Reply:
There was an error while running black that it tries to execute it. But i think i made a mistake by running black *

Reply:
Yes, probably you should try with `black .`